### PR TITLE
Refresh

### PR DESCRIPTION
**Pull request by ocefpaf**

### CODE REVIEW

**Comment by ocefpaf** [comment]
```
[conda] C:\conda\conda-bld\test-tmp_dir>call activate.bat C:\conda\envs\_test 
===== testing package: mplleaflet-0.0.5-py27_0 =====
import: u'mplleaflet'
Traceback (most recent call last):
  File "C:\conda\conda-bld\test-tmp_dir\run_test.py", line 26, in <module>
    import mplleaflet
  File "C:\conda\envs\_test\lib\site-packages\mplleaflet\__init__.py", line 3, in <module>
    from mplleaflet._display import (
  File "C:\conda\envs\_test\lib\site-packages\mplleaflet\_display.py", line 10, in <module>
    import matplotlib.pyplot as plt
  File "C:\conda\envs\_test\lib\site-packages\matplotlib\pyplot.py", line 114, in <module>
    _backend_mod, new_figure_manager, draw_if_interactive, _show = pylab_setup()
  File "C:\conda\envs\_test\lib\site-packages\matplotlib\backends\__init__.py", line 32, in pylab_setup
    globals(),locals(),[backend_name],0)
  File "C:\conda\envs\_test\lib\site-packages\matplotlib\backends\backend_qt4agg.py", line 18, in <module>
    from .backend_qt5agg import FigureCanvasQTAggBase as _FigureCanvasQTAggBase
  File "C:\conda\envs\_test\lib\site-packages\matplotlib\backends\backend_qt5agg.py", line 16, in <module>
    from .backend_qt5 import QtCore
  File "C:\conda\envs\_test\lib\site-packages\matplotlib\backends\backend_qt5.py", line 31, in <module>
    from .qt_compat import QtCore, QtGui, QtWidgets, _getSaveFileName, __version__
  File "C:\conda\envs\_test\lib\site-packages\matplotlib\backends\qt_compat.py", line 137, in <module>
    from PyQt4 import QtCore, QtGui
ImportError: No module named PyQ
```

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,37 +1,38 @@
...
-    url: https://pypi.python.org/packages/source/m/mplleaflet/mplleaflet-{{ version }}.tar.gz
-    md5: cf8ec819862e4486f477fb71101477f1
+  fn: mplleaflet-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/m/mplleaflet/mplleaflet-{{ version }}.tar.gz
+  sha256: 049e0b91797ce5b462853395138161fed9e8dfc1f4723f482ebb0739a0bbd289
 
 build:
-    number: 0
-    script: python setup.py install --single-version-externally-managed --record record.txt
+  number: 0
```
Comment (jakirkham) [inline recipe/meta.yaml]: Build number bump (given run time dependencies have changed)?

**Comment by ocefpaf** [comment]
For some reason, even with the latest conda and channel preferences, conda is still getting some packages from `defaults`. That is a bummer and let's hope that over-specifying `matplotlib` as a dependency here solves this.

``` shell
    ca-certificates: 2016.8.31-0       conda-forge
    certifi:         2016.8.31-py27_0  conda-forge
    cycler:          0.10.0-py27_0     conda-forge
    freetype:        2.6.3-vc9_1       conda-forge [vc9]
    functools32:     3.2.3.2-py27_1    conda-forge
    icu:             57.1-vc9_0        defaults    [vc9]
    jinja2:          2.8-py27_1        conda-forge
    jpeg:            8d-vc9_2          defaults    [vc9]
    libpng:          1.6.24-vc9_0      conda-forge [vc9]
    markupsafe:      0.23-py27_0       conda-forge
    matplotlib:      1.5.2-np111py27_5 conda-forge
    mkl:             11.3.3-1          defaults   
    mplexporter:     0.0.1-py27_1      conda-forge
    mplleaflet:      0.0.5-py27_0      local      
    numpy:           1.11.2-py27_0     defaults   
    openssl:         1.0.2h-vc9_2      conda-forge [vc9]
    pyparsing:       2.1.9-py27_0      conda-forge
    pyqt:            5.6.0-py27_0      defaults   
    python:          2.7.12-1          conda-forge
    python-dateutil: 2.5.3-py27_0      conda-forge
    pytz:            2016.7-py27_0     conda-forge
    qt:              5.6.0-vc9_0       defaults    [vc9]
    setuptools:      26.1.1-py27_0     conda-forge
    sip:             4.18-py27_0       conda-forge
    six:             1.10.0-py27_0     conda-forge
    vs2008_runtime:  9.00.30729.1-2    defaults   
    wincertstore:    0.2-py27_0        conda-forge
    zlib:            1.2.8-vc9_3       conda-forge [vc9]
```

**Comment by ocefpaf** [comment]
OK. bumping `matplotlib` in the dependency priority by over-specifying it here works :unamused: 

I will remove `mplexporter` as latest `mplleaflet` ships its own copy and specify more dependencies in this feedstock then.

**Pull request merged**